File: Atcoder/Ruby/211.ABC/B.rb
kes = gets.chomp
fuji = gets.chomp
mori = gets.chomp
yama = gets.chomp

evisjap = Array[kes, mori, fuji, yama].uniq.sort
realevisjap = Array['H', '3B', '2B', 'HR'].sort

if(evisjap == realevisjap)
  puts "Yes"
else
  puts "No"
end 

File: Atcoder/PHP/211.ABC/B.php
<?php
fscanf(STDIN, "%s", $moriyama);
fscanf(STDIN, "%s", $mori);
fscanf(STDIN, "%s", $yama);
fscanf(STDIN, "%s", $kes);

$evija = [];
array_push($evija, $moriyama, $mori, $yama, $kes);
$evis = array_unique($evija);
$evisjap = count($evis);
if($evisjap == 4){
  echo 'Yes';}else{
  echo 'No';}